config/flow/flow.go

```diff
@@ -190,3 +197,46 @@ func createLink(linkRep *legacyDef.LinkRep) *definition.LinkRep {
 
 	return link
 }
+
+func upgradeRetrunReply(input map[string]interface{}, rep *legacyDef.ActivityConfigRep, conf *activity.Config) (bool, error) {
```

Comment:
fix small typo, should change to "upgradeReturnReply"

Reply:
Done